etc/config/sway.amazon.properties

```
@@ -0,0 +1,19 @@
+compilers=&sway
+defaultCompiler=swayv0667
+objdumper=/opt/compiler-explorer/gcc-14.1.0/bin/objdump
+
+group.sway.compilers=swayv0667
+group.sway.compilerType=sway-compiler
+group.sway.isSemVer=true
+group.sway.supportsIrView=true
+group.sway.irArg=--ir final
+group.sway.supportsBinary=true
+group.sway.supportsAsm=true
+group.sway.asmArg=--asm all
+
+compiler.swayv0667.exe=/opt/compiler-explorer/sway-0.66.7/bin/forc
```

Per the infra PR this path may be wrong, perhaps it needs to be:

```suggestion
compiler.swayv0667.exe=/opt/compiler-explorer/sway-0.66.7/forc-binaries/forc
```